team-policy.md

```diff
@@ -0,0 +1,138 @@
+# Style Team Policy Document
+
+This is a living document tracking the active policies of the style team. It is intended to fill a similar role to the books that many[^1] teams[^2] maintain[^3] independently[^4] with the intent that eventually these can be integrated into a central policy repository for all teams in the rust-lang organization.
+
+* **Original RFC**: https://rust-lang.github.io/rfcs/3309-style-team.html
+* **Charter**: TODO
+
+## Operational Roles
+
+The style team has various operational roles to delegate releated and ongoing work to specific individuals, to clarify on our operations, and to coordinate sharing feedback with the members in those roles to enable iterative self improvement.
+
+### Operational Lead
+
+* Current Holder: [@calebcartwright](https://github.com/calebcartwright)
+* Next Feedback Date: TODO
+
+In order to stay in touch with where we want to be heading in the future, we need leadership. A team leader is paying attention to the team’s operations in relation to the team’s aim. What needs to be done, who agreed to do it. What is in the future to decide?
+
+Responsibilities:
+
+* Pays attention to the operation of the team
+* Pays attention to the team members
+* Reports wider context to the team
+
+### Facilitator
+
+* Current Holder: [@yaahc](https://github.com/yaahc)
+* Next Feedback Date: Wednesday 2023-02-01
+
+In order to be present with each other, we need a good facilitator .  Facilitators run meetings according to the format of meetings and decision making adopted by the group. Leader and facilitator are separate roles because facilitation and overseeing operations are different skill sets. They can be held by the same individual.
```

Comment:
Minor item, but I think we're only planning on having a single facilitator and that it would be good to use the singular form to avoid any potential misinterpretations of there being multiple 

```suggestion
In order to be present with each other, we need a good facilitator.  The facilitator runs meetings according to the format of meetings and decision making adopted by the group. Leader and facilitator are separate roles because facilitation and overseeing operations are different skill sets. They can be held by the same individual.
```